### PR TITLE
0.21.1 backports (round 1)

### PR DESCRIPTION
**Pull request by TomAugspurger**
Reminder: don't squash before merging.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
@jreback @jorisvandenbossche the failure in https://travis-ci.org/pandas-dev/pandas/jobs/313729058 is our old friend, the clipboard failures, since https://github.com/pandas-dev/pandas/pull/17723 wasn't backported.

This should be good to go, and then I'll need to make sure I didn't miss any on this round. I'll merge later today.

**Comment by jreback** [comment]
hmm i though i has a commit where i moves the clipboard tests

but oh well

File: doc/source/whatsnew/v0.21.1.txt [hunk 1]
```diff
@@ -54,40 +83,57 @@ Documentation Changes
 
 .. _whatsnew_0211.bug_fixes:
 
-Bug Fixes
```
Comment (jreback) [inline doc/source/whatsnew/v0.21.1.txt]: I think we need a section header here  for all the bug fixes (can do in followup)

**Review by jreback — APPROVED** [review]
very minor doc issue. otherwise lgtm.

File: doc/source/whatsnew/v0.21.1.txt (hunk 1 above)
Comment (jorisvandenbossche) [reply doc/source/whatsnew/v0.21.1.txt]: That should go in a normal PR to master, we should not do any fixes here (apart from possible merge conflicts)

Comment (TomAugspurger) [reply doc/source/whatsnew/v0.21.1.txt]: Ah, that was my fault. I messed up a merge conflict I think when cherry picking.

Comment (jorisvandenbossche) [reply doc/source/whatsnew/v0.21.1.txt]: ah, yes, this one indeed seems like a wrong merge

**Comment by jorisvandenbossche** [comment]
@TomAugspurger This can be merged? (the failure is a clipboard related, so I think can be ignored)

**Comment by TomAugspurger** [comment]
Oh, yes. I thought I had.

**Pull request merged**